Tools/electrostatic_sphere_plot.py

```diff
@@ -0,0 +1,149 @@
+import yt
+import matplotlib.pyplot as plt
+from scipy.optimize import fsolve
+from skimage import measure
+import numpy as np
+
+yt.funcs.mylog.setLevel(50)    #Mute yt's verbose shenanigans
+plotfilepath = '../Examples/Tests/ElectrostaticSphere/diags/plotfiles'
```

Comment:
If you place it alongside the input script you can change the default here (e.g. to the local directory)